jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -288,4 +298,8 @@ private Variant loadVariantWhenAuthorized(
         // unauthorized...
         return null;
     }
+
+    private String extractPushIdentifier(HttpServletRequest request) {
+        return request.getHeader("push-identifier");
```

Comment:
@sebastienblanc It should be a header, since we should not really expose this hook on the metadata for the device registry